tests/smoke-tests/src/main/resources/servers/audit-logging2/log4j2.properties

```diff
@@ -30,6 +30,9 @@ logger.artemis_journal.level=INFO
 logger.artemis_utils.name=org.apache.activemq.artemis.utils
 logger.artemis_utils.level=INFO
 
+logger.StompConnection.name=org.apache.activemq.artemis.core.protocol.stomp.StompConnection
+logger.StompConnection.level=TRACE
+
```

Comment:
Is this still needed if the new tests moved to AuditLoggerResourceTest? (looks to use different server config)

Reply:
I left this in by mistake.